python/cdp_montana_legislature_backend/scraper.py

```diff
@@ -1,11 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import logging
 from datetime import datetime
 from typing import List
 
 from cdp_backend.pipeline.ingestion_models import EventIngestionModel
 
+# region logging
+
+log = logging.getLogger("cdp_montana_legislature_scraper")
+log.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.DEBUG)
+ch.setFormatter(
+    logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+)
+log.addHandler(ch)
```

Comment:
Note: I think this will override the default logger set for the pipeline in "production". My recommendation would be to move this into the __main__.

Reply:
the `log = ` line is fine. its the `setLevel` calls that override the default logger for production

Reply:
Ah! You are totally right. Fixed this in the branch now.